src/include/snapper/dialogs.rb

```diff
@@ -25,12 +25,9 @@
 # Authors:	Jiri Suchomel <jsuchome@suse.cz>
 
 module Yast
-
```

Comment:
bbatsov is autohor of rubocop and his style-guide is semi official ruby style guide. I see that our style guide do not mention it, but when we discuss it conclusion was that we follow this widely known guide and mention only our specific extension to style guide. I will open pull request for our style guide to reflect it.